### ISSUE TITLE
How to use built in SSML support?

### ISSUE THREAD

**Issue by BrianRosamilia** [alexa-js/alexa-app#74]
Greetings,

Sorry, but I am unclear on how SSML support works currently

The only success I've had in getting an SSML output is by doing the following

```
awsResponse.response.response.outputSpeech.type = 'SSML';
awsResponse.response.response.outputSpeech.ssml = awsResponse.response.response.outputSpeech.text;
```

And manually inserting my own 'say' tags.

Obviously this isn't the ideal way to do it, but there I'm a bit confused on how to actually get an SSML response to work.

No matter what I put in response.say currently, the response type is always 'PlainText'.  I noticed that calling `clear` can lead to this behavior, but I never call it.

If it makes a difference, my intent and launch handlers are asynchronous

**Comment by dblock** [comment]
Have you resolved this @BrianRosamilia?

**Comment by tejashah88** [comment]
@BrianRosamilia You might want to consider the [ssml-builder](https://github.com/mandnyc/ssml-builder) library.

**Comment by dblock** [comment]
I think we need a better example using ssml-builder in README and close this.

**Comment by tejashah88** [comment]
Resolved with #276
